src/eth/EthereumCdpService.js

```diff
@@ -105,8 +106,13 @@ export default class EthereumCdpService extends PrivateService {
     });
   }
 
-  async lockEth(cdpId, amount, unit = ETH) {
-    await this._conversionService().convertEthToWeth(amount, unit);
+  @tracksTransactions
```

Comment:
this is "decorator" syntax -- see this [tutorial](https://www.sitepoint.com/javascript-decorators-what-they-are/) and the implementation in TransactionManager

Reply:
the changes to .babelrc, .eslintrc, webpack.config.js are so that the linter and builds (both frontend & backend) know how to handle the syntax